system/include/webgl/webgl1_ext.h

```diff
@@ -228,6 +228,16 @@ WEBGL_APICALL void GL_APIENTRY emscripten_webgl_getSupportedAstcProfiles(GLsizei
 #define GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT3_EXT 0x8C4E
 #define GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT5_EXT 0x8C4F
 
+// 40. https://www.khronos.org/registry/webgl/extensions/WEBGL_multi_draw/
+WEBGL_APICALL void GL_APIENTRY emscripten_glMultiDrawArraysWEBGL(GLenum mode, const GLint* firstsList, GLuint firstsOffset, const GLsizei* countsList, GLuint countsOffset, GLsizei drawcount);
```

Comment:
These signatures need to be revised. Please compare to the native ANGLE extension:
https://chromium.googlesource.com/angle/angle/+/master/extensions/ANGLE_multi_draw.txt

for the WebGL multi-draw extension:
https://www.khronos.org/registry/webgl/extensions/WEBGL_multi_draw/

All of the JavaScript argument pairs like "firstsList / firstsOffset" should just be a single pointer - "GLint* firsts" - in the C signature.

The underlying bindings then pass the incoming pointer from WASM - which is just an offset into the heap - down into the JavaScript entry point, without creating any temporary typed arrays.

Another question that needs to be answered: should these be named according to the underlying ANGLE extension? In this case, emscripten_glMultiDrawArraysANGLE? If C code were being written and linking against ANGLE then it would be calling glMultiDrawArraysANGLE, so I think probably these entry points should be named with the ANGLE suffix rather than the WEBGL suffix.